### ISSUE TITLE
Prepare with Ddan and Durgadas two awesome Gravity Wg calls :) for 17th & 24th June

### ISSUE THREAD

**Issue by JuankBell** [CommonsBuild/coordination#728]
## 👇🏽  What is this issue about? Give as much detail as you can. 

@JuankBell will be traveling to Perú from the 14th to 28 of June. He will be following the community but this is a perfect opportunity to bring fresh air to the Gravity Wg calls and empower Gravitons to add value to the sessions. @ddan4 volunteered to do this and lead this meetings for two weeks, so we have full confidence he will do great, but it would be good to have something prepared. This sprint we can have one or two hack sessions for this.

## ✨  What does success look like? (i.e., the criteria to meet to call this issue done)

1. Gravity gaining fresh air having new value added to the sessions
2. Juan taking time to recharge 
3. Empowering Gravitons

##🧍🏻‍♀️🧍🏻‍♂️ Who should be part of the Advice Process? (People who should be consulted about this issue. Those who will have valuable input.)

@liviade1 @ddan4 @JuankBell 

## 👀  Who should be in the loop about this issue?
@GriffGreen @durgadas

**Comment by ddan4** [comment]
I have a couple ideas for Gravity sessions.. but realised I will be available for only the first session on the 14th.

One idea think is most approachable for the near term would be exploring TEC praise through an NVC model, which people could use to practice NVC thought-forms as part of a practice we're already keeping

Its a simple concept I think could be really valuable to explore as a group

TLDR; 

!Praise @[...] for doing [...], which I felt [...], having my need for [...] met

Alternatively,  I have some notes I've been pulling together from the lectures of Marshall Rosenberg I'm interested to make a reasonably comprehensive one-pager of his teaching that would ideally make it fairly digestible 

The second option is more work, overall... but definitely doable if that's preferable, really if I finish what I started note-taking on, it would probably provide enough material for 2 sessions, and would likely be better appreciated than crammed into one.

if we really like that idea, then I can pull those notes into shareable form sooner than later.

**Comment by JuankBell** [comment]
Thanks Ddan, the one pager of Marshall Rosenberg seems a great idea. My initial thinking is that we can prepare this documentation without rush for the second Graviton training, and that we could schedule 2 sessions to cover this topic.

For the session of the 14th I would go for more approachable focus. Following the temperature of the community for the Hatch (which will be near these weeks), discussing about NVC models and practices, & talking about strategies for choosing Celeste's params as a community for the implementation of disputable voting.

- (this last idea came after May 30th params call where it was considered to take away Celeste from the Commons upgrade dashboard and make it more a community decision)

**Comment by JuankBell** [comment]
And for 24th June, will try to sync with Durgadas to see if he would be willing to lead that day.

**Comment by JuankBell** [comment]
Ddan Will take 17th with a NVC workshop, and Durgadas will tale the 24th for a Decolonization workshop :D Really cool sessions!

**Comment by JuankBell** [comment]
Ddan rocked it last week and today's Session lead by Durgadas will be surely great! Closing this issue :) thanks for all your support.
